<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-DGDCJGPO" failures="1" tests="1" name="testCaseHSTL.TC14_HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong" time="174.357" errors="0" timestamp="2021-10-30T17:32:53 ICT" skipped="0">
  <testcase classname="testCaseHSTL.TC14_HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong" name="HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong" time="174.357">
    <failure message="expected [Yêu cầu phê duyệt &quot;[VTTest] Hồ sơ tài liệu ký Mộc - Chuyển xử lý ở Bước Trưởng phòng 30-10-2021 05:59&quot;] but found [Chia sẻ dự thảo &quot;[VTTest] TC 11 - Chuyển xử lý tại bước Lãnh đạo Ban Giám đốc 1635584884433&quot;]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Yêu cầu phê duyệt "[VTTest] Hồ sơ tài liệu ký Mộc - Chuyển xử lý ở Bước Trưởng phòng 30-10-2021 05:59"] but found [Chia sẻ dự thảo "[VTTest] TC 11 - Chuyển xử lý tại bước Lãnh đạo Ban Giám đốc 1635584884433"]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pages.DanhSachViecCanXuLy.ValidateTaskYeuCauPheDuyetHopDong(DanhSachViecCanXuLy.java:288)
at testCaseHSTL.TC14_HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong.HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong(TC14_HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong -->
  <system-out>
    <![CDATA[http://eoffice.petrolimex.com.vn/plxdb/vanban/SitePages/VanBanDi_Detail.aspx?IId=5413&SPId=64&ListID=9710b5f7-0241-4714-8041-0f596f9ae605]]>
  </system-out>
</testsuite> <!-- testCaseHSTL.TC14_HoSoTaiLieuKyMocChuyenXuLyBuocTruongPhong -->
